controllers/comments.js

```diff
@@ -0,0 +1,16 @@
+const {findCommentsByArticle} = require('../models/comments')
+const {checkArticleExists} = require('../models/articles')
+
+function getCommentsByArticle(req,res,next) {
+  const {article_id} = req.params
+  Promise.all([findCommentsByArticle(article_id), checkArticleExists(article_id)])
```

Comment:
Great solution here to handle the empty response

Reply:
Thanks, can't take all the credit. It's how David showed us in the lecture.